test/gallery_screen_test.mocks.dart

```diff
@@ -105,8 +105,6 @@ class MockGalleryProvider extends _i1.Mock implements _i4.GalleryProvider {
       (super.noSuchMethod(Invocation.method(#deleteRequest, [url, id]),
           returnValue: Future<_i3.Response>.value(_FakeResponse_3())) as _i6.Future<_i3.Response>);
   @override
-  String toString() => super.toString();
```

Comment:
FYI, all the foo.mocks.dart files are generated, so this will come back the next time we start the build runner

Reply:
Ohh thanks I completely missed that, haven't really looked at the file names